src/modules/commander/rc_calibration.cpp

```diff
@@ -65,34 +66,37 @@ int do_trim_calibration(orb_advert_t *mavlink_log_pub)
 	sub_man.copy(&sp);
 
 	/* load trim values which are active */
-	float roll_trim_active;
-	param_get(param_find("TRIM_ROLL"), &roll_trim_active);
-	float pitch_trim_active;
-	param_get(param_find("TRIM_PITCH"), &pitch_trim_active);
-	float yaw_trim_active;
-	param_get(param_find("TRIM_YAW"), &yaw_trim_active);
+	param_t param_trim_roll = param_handle(px4::params::TRIM_ROLL);
+	param_t param_trim_pitch = param_handle(px4::params::TRIM_PITCH);
+	param_t param_trim_yaw = param_handle(px4::params::TRIM_YAW);
+	param_t param_fw_man_r_sc = param_handle(px4::params::FW_MAN_R_SC);
+	param_t param_fw_man_p_sc = param_handle(px4::params::FW_MAN_P_SC);
+	param_t param_fw_man_y_sc = param_handle(px4::params::FW_MAN_Y_SC);
+
+	float roll_trim_active, pitch_trim_active, yaw_trim_active, roll_scale, pitch_scale, yaw_scale;
```

Comment:
It's probably a bit pedantic here, but I think it's good practice to declare and initialize each of these on a separate line with a sane default value.

Reply:
@dagar the issue here is not mainly about the searching of the parameter, but about the problem that the rc_calibration search for parameter that doesn't exists, on some compilation